tachyon/zk/lookup/log_derivative_halo2/prover.h

```diff
@@ -59,8 +59,8 @@ struct GrandSumPolysTempStorage {
   std::vector<F> table_log_derivatives;
 
   explicit GrandSumPolysTempStorage(size_t usable_rows)
-      : inputs_log_derivatives(usable_rows),
-        table_log_derivatives(usable_rows) {}
+      : inputs_log_derivatives(usable_rows, F::Zero()),
+        table_log_derivatives(usable_rows, F::Zero()) {}
```

Comment:
It seems like `CreateGrandSumPoly` fills the values. But just add a comment that why this is safe.